changelog/unreleased/bugfixes/7753.md

```diff
@@ -0,0 +1,4 @@
+- Fix adasis generator errors related to the split edges.
+- Fix adas cache tiles eviction mechanism.
+- Handle "unlilimited" speed limits in Adasis.
+- Fixed possible `java.lang.UnsatisfiedLinkError` crash on application startup when Mapbox native libraries could not yet be found.
```

Comment:
```suggestion
- Fix possible `java.lang.UnsatisfiedLinkError` crash on application startup when Mapbox native libraries could not yet be found.
```
to align with other records

Reply:
We actually use past simple in our changelog.